etc/examples/nodes/fpga.conf

```
@@ -19,8 +19,9 @@ fpgas = {
 nodes = {
     fpga_0 = {
         type = "fpga",
-        card = "vc707"
-        connect = ["0->3", "3->dma", "0<-dma"]
+        card = "vc707",
+        connect = ["0->3", "3->dma", "0<-dma"],
```

I know its unrelated to this PR, but just a recommendation for improving the config schema:

```suggestion
        connect = (
          { master = 0,     slave = 3     },
          { master = 3,     slave = "dma" },
          { master = "dma", slave = 0     }
        ),
```

`master`, `slave` could also be replaced with `from`, `to`.

This would more closer match the `paths` setting of VILLASnode itself:

```
paths = (
  {
    in = "node1"
    out = "node2"
  }
)
```

---

yes makes sense. Will consider it.